gulpfile.js

```diff
@@ -220,14 +220,16 @@ gulp.task('watch', function () {
     'modules/**/*.js',
     'test/spec/**/*.js',
     '!test/spec/loaders/**/*.js'
-  ], ['lint', 'build-bundle-dev', 'test']);
+  // ], ['lint', 'build-bundle-dev', 'test']);
+  ], ['build-bundle-dev']);
   gulp.watch([
     'loaders/**/*.js',
     'test/spec/loaders/**/*.js'
   ], ['lint']);
   connect.server({
+    host: '0.0.0.0',
     https: argv.https,
-    port: port,
+    // port: '8080',
```

Comment:
We shouldn't be committing any changes in this file to Prebid/Prebid.js - I guess @jzepeda-ias based it off my develop branch, where I had these changes purely for easier local development.